test/Concurrency/sending_interfacefile_printing.swift

```diff
@@ -0,0 +1,23 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend %s -typecheck -enable-library-evolution -parse-as-library -emit-module-interface-path - -module-name MyFile -swift-version 6 | %FileCheck -DFILE_PREFIX=MyFile %s
```

Comment:
Is the `-DFILE_PREFIX=MyFile` used here?

Reply:
You are right... I removed it.